README.md

```diff
@@ -696,8 +692,8 @@ Always use semicolons whenever they are syntactically allowed. For example:
 if(a);
 else;
 {
-int d;
-d = c;
+ int d;
+ d = c;
```

Comment:
I would ask you revert this one since it's so unorthodox. It's nearly impossible for me to discern what constitutes good style in this context of oppressive semicolon ~ab~use, so let's just keep it all unindented I believe it helps add to the joke.